### ISSUE TITLE
Multiple consoles connected to PegaSwitch

### ISSUE THREAD

**Issue by misson20000** [reswitched/pegaswitch#41]
I think it'd be cool to be able to connect multiple Switches to PegaSwitch at once. Give each one a name, include it in the prompt, and have a command to change which console is "active". This requires some method of uniquely identifying the console, assigning a name, and keeping track of these assignments. [set:sys#68 GetSerialNumber()](https://reswitched.github.io/SwIPC/ifaces.html#nn::settings::ISystemSettingsServer) might be a good choice for the unique ID.

**Comment by JoelLarson** [comment]
I'll start tackling this. Will it be a concern to use serial number in an identifiable way?

I could see this being useful for someone mounting on the Internet and anyone being able to set their DNS to the given IP without needing to understand pega. I'm not sure if that is the intent of this feature though.

**Comment by SomeoneWeird** [comment]
We already run a public DNS which blocks Nintendo updates/error reporting. You should never ever expose your Pegaswitch instance to the internet :)

**Comment by JoelLarson** [comment]
@SomeoneWeird it seems the feature is more for people wanting to run things against multiple consoles at one time without having to do crazy things

**Comment by SomeoneWeird** [comment]
Yes, I know, I was responding to the second paragraph in your previous comment where you alluded to running pegaswitch on the internet :)

**Comment by misson20000** [comment]
Usage example:
![screenshot](https://i.imgur.com/KT6T5Eo.png)
How does it look? Anything I should change? Assigning names to consoles is not yet implemented.
